test/e2e/specs/nux.test.js

```diff
@@ -35,13 +32,7 @@ describe( 'New User Experience (NUX)', () => {
 	}
 
 	beforeEach( async () => {
-		await newDesktopBrowserPage();
-		await newPost( undefined, false );
-	} );
-
-	afterEach( async () => {
-		// Clear localStorage tips so they aren't persisted for the next test.
-		await clearLocalStorage();
```

Comment:
Is this not needed anymore?

Reply:
We clear `localStorage` after every test suite. In addition, when opening the editor and `enableTips` flag is passed we clean all disabled tips.